src/mudlet.cpp

```diff
@@ -3410,6 +3429,61 @@ void mudlet::slot_irc()
     mpIrcClientMap.value(pHost)->show();
 }
 
+void mudlet::slot_discord()
+{
+    openWebPage(mMudletDiscordInvite);
+}
+
+void mudlet::updateMudletDiscordInvite()
+{
+    QDir dir;
+    QString cacheDir = QStandardPaths::writableLocation(QStandardPaths::CacheLocation);
+    if (!dir.mkpath(cacheDir)) {
+        qWarning() << "Couldn't create cache directory for Mudlet's Discord JSON file: " << cacheDir;
+        return;
+    }
+
+    auto manager = new QNetworkAccessManager(this);
```

Comment:
Not *another* `QNetworkAccessManager` instance - the Qt Documentation says that one is enough for a whole application yet we create two or three of these for each profile. :roll_eyes:

Reply:
I'll consolidate them in a follow-up PR.

Reply:
I've spent some time looking at this, but my C++ isn't strong enough to figure out a good way to do this.